sp800-63a/sec5_proofing.md

```diff
@@ -85,8 +85,11 @@ The goal of identity validation is to establish a linkage to the physical, live
 |:---:|:------------------------------| 
 |Unacceptable|Evidence verification was not performed, or verification of the evidence failed. Unable to confirm that the applicant is the owner of the claimed identity.|
 |Weak|The applicant has been confirmed as having access to the evidence provided to support the claimed identity.|
-|Adequate|- The applicant’s ownership of the claimed identity has been confirmed by KBV.  See [Section 5.3.2](#kbv)  for more details.<br>**OR**<br>- The applicant’s ownership of the claimed identity has been confirmed by a physical **OR** biometric comparison of the applicant to the strongest piece of evidence provided. Physical comparison performed remotely requires liveness detection as specified in [Section 5.2.3](#biometric_use) of [SP 800-63B](#800-63b). Biometric comparison performed remotely all appropriate requirements as specified in [Section 5.2.3](#biometric_use) of [SP 800-63B](#800-63b) to be met. |
-|Strong|- The applicant’s ownership of the claimed identity has been confirmed by physical comparison, using appropriate equipment, to a photograph/image. Physical comparison performed remotely requires liveness detection as specified in [Section 5.2.3](#biometric_use) of [SP 800-63B](#800-63b). <br>**OR**<br>- Biometric comparison, using appropriate equipment, of the applicant to the strongest piece of evidence provided to support the claimed identity. Biometric comparison performed remotely all appropriate requirements as specified in [Section 5.2.3](#biometric_use) of [SP 800-63B](#800-63b) to be met.|
+|Adequate|- The applicant’s ownership of the claimed identity has been confirmed by KBV.  See [Section 5.3.2](#kbv)  for more details.<br>**OR**<br>- The applicant’s ownership of the claimed identity has been confirmed by a physical **OR** biometric comparison of the applicant to the strongest piece of evidence provided. Physical comparison performed remotely should include presentation attack detection as specified in [Section 5.2.3](#biometric_use) of [SP 800-63B](#800-63b). Biometric comparison performed remotely shall adhere to the
+
+
```

Comment:
This and the previous blank line appear to be accidental and will mess up the table.